config.py

```diff
@@ -133,7 +134,10 @@ class TestingConfig(BaseConfig):
     MOCK_EMAIL = True
 
     # Use in-memory SQLite database for testing
-    SQLALCHEMY_DATABASE_URI = "sqlite://"
+    # SQLALCHEMY_DATABASE_URI = "sqlite://"
+    SQLALCHEMY_DATABASE_URI = BaseConfig.build_db_uri(
```

Comment:
this will still work on docker, even if I don't have Postgres locally, right?

Reply:
Yes. Docker will fetch the postgres image for you.